src/utils/editor/tests/get-expression.spec.js

```diff
@@ -0,0 +1,175 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
+
+import CodeMirror from "codemirror";
+import { getExpressionFromCoords } from "../get-expression";
+
+describe("get-expression", () => {
+  let isCreateTextRangeDefined;
+
+  beforeAll(() => {
+    if (document.body.createTextRange) {
+      isCreateTextRangeDefined = true;
+    } else {
+      isCreateTextRangeDefined = false;
+      // CodeMirror needs createTextRange
+      // https://discuss.codemirror.net/t/working-in-jsdom-or-node-js-natively/138/5
+      document.body.createTextRange = () => ({
+        getBoundingClientRect: () => {},
+        getClientRects: () => ({})
+      });
```

Comment:
does jest have a `before` block that is only run once? I'd like to avoid `isCreateTextRangeDefined`

Reply:
No, I believe if you only need it once you would write it in the top of the `test` body. Or if it's needed in all the tests but only needs to be created once you can do it at the top of the `describe` or module if I'm recalling right.

Reply:
I think that `document.body` is a global variable or something like that so if I added `createTextRange` to it I wanted to delete it at the end of these tests so it wouldn't affect other tests.  Is there another way to keep track of whether or not I should delete `createTextRangeDefined` at the end of the tests?